<ui version="4.0" >
 <class>HistoryMsg</class>
 <widget class="QDialog" name="HistoryMsg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="styleSheet" >
   <string>QDialog {
 
background-color:rgb(246,243,243);

}
</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="MsgBrowser" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>40</y>
     <width>471</width>
     <height>211</height>
    </rect>
   </property>
   <property name="focusPolicy" >
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="lineWidth" >
    <number>2</number>
   </property>
   <property name="verticalScrollBarPolicy" >
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="autoFormatting" >
    <set>QTextEdit::AutoAll</set>
   </property>
   <property name="undoRedoEnabled" >
    <bool>true</bool>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
   <property name="html" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif';">&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif';">&lt;/p>&lt;/body>&lt;/html></string>
   </property>
   <property name="overwriteMode" >
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags" >
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="todayBT" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>today</string>
   </property>
  </widget>
  <widget class="QPushButton" name="oneweekBT" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>10</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>one week</string>
   </property>
   <property name="shortcut" >
    <string>PgDown</string>
   </property>
  </widget>
  <widget class="QPushButton" name="twoweekBT" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>10</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>two week</string>
   </property>
  </widget>
  <widget class="QPushButton" name="onemonthBT" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>10</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>one month</string>
   </property>
  </widget>
  <widget class="QPushButton" name="allBT" >
   <property name="geometry" >
    <rect>
     <x>370</x>
     <y>10</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>all</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HistoryMsg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HistoryMsg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
